public/ch.json

```diff
@@ -286,7 +286,8 @@
     "security_screen": "安全屏幕",
     "enter_pass": "填写您的密码",
     "_pass": "密码",
-    "pass_from_env": "填写.env文件中定义的密码"
+    "pass_from_env": "填写.env文件中定义的密码",
+    "connect": "Connect"
```

Comment:
A tip, I had problems translating the .json to other languages when the variable name is a common word like connect, it could translate it to the other language and increase the work needed to translate it.

Reply:
You say that i have to use '' common '' part when its an only and reused word, right?